modules/abstract-eth/package.json

```diff
@@ -41,6 +41,8 @@
     "@bitgo/sdk-core": "^7.0.1",
     "@bitgo/statics": "^14.4.0",
     "@bitgo/utxo-lib": "^7.7.0",
-    "bignumber.js": "^8.0.1"
+    "bignumber.js": "^8.0.1",
+    "@types/keccak": "^3.0.1",
```

Comment:
`@types/keccak` should be a dev dependency

Reply:
not required after removing default implementation of getHashFunction in abstractEth